pipelines/broad/arrays/single_sample/Arrays.documentation.md

```diff
@@ -26,52 +33,108 @@ The Arrays Pipeline takes the inputs described below. Inputs are grouped by func
-    *   extended_chip_manifest_file. The cloud path of the Illumina chip manifest for the genotyping array to be processed, in CSV form, as extended by the Picard tool CreateExtendedIlluminaManifest.
-    *   cluster_file. The cloud path of the Illumina cluster file (.egt) for the genotyping array to be processed.
-    *   gender_cluster_file [Optional]. The cloud path of the Illumina gender cluster file (.egt) for the genotyping array to be processed. This optional cluster file is used to run IAAP/gencall specifically for calling the gender of the sample.  It typically is designed with all assays other than those at specific sites on the sex chromosomes zeroed out.
-    *   zcall_thresholds_file [Optional]. The cloud path of a thresholds file to be used with zCall. [zCall](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3463112/) is a rare variants caller that can be used to improve the calls on rare variant assays on the genotyping array. If this optional parameter is provided, the pipeline will run zCall using the specified thresholds file.
+    *   arrays_metadata_path [Optional]. The cloud path where chip-specific metadata will be looked for by the workflow under certain conditions as explained below:
+
+    *   bead_pool_manifest. The cloud path of the Illumina bead pool manifest file (.bpm) that will be used to process the array data is determined based on which of the following inputs are provided:
+
+        *   bead_pool_manifest_filename [Optional]. The **name** of the Illumina bead pool manifest file that will be used to process the array data. If specified, a file by this name will be looked for in arrays_metadata_path. If **NOT** specified, the pipeline will use the **bead_pool_manifest_file**.
+
+        *   bead_pool_manifest_file [Optional]. The cloud path of the Illumina bead pool manifest file for the genotyping array to be processed.
+
+    *   cluster. The cloud path of the Illumina cluster file (.egt) that will be used to process the array is determined based on which of the following inputs are provided:
+
+        *   cluster_filename [Optional]. The **name** of the Illumina cluster file (.egt) that will be used to process the array data. If specified, a file by this name will be looked for in arrays_metadata_path. If **NOT** specified, the pipeline will use the **cluster_file**.
+
+        *   cluster_file [Optional]. The cloud path of the Illumina cluster file (.egt) for the genotyping array to be processed.
+
+    *   gender_cluster [Optional]. The gender cluster file is an Illumina cluster file (.egt) that is used to run IAAP/gencall specifically for calling the gender of the sample.  It typically is designed with all assays other than those at specific sites on the sex chromosomes zeroed out. The cloud path of the gender_cluster_file that is used is determined based on which of the following inputs are provided:   
```

Comment:
Is this still an input to the pipeline? I am only seeing `gender_cluster_filename` and `gender_cluster_file`